Docs: added clarification to plugin provisioning (#50212)
grafana/grafana: docs/sources/administration/provisioning.md

> This feature is available from v7.1

> You can manage plugin applications in Grafana by adding one or more YAML config files in the [`provisioning/plugins`]({{< relref "../setup-grafana/configure-grafana/#provisioning" >}}) directory. Each config file can contain a list of `apps` that will be updated during start up. Grafana updates each app to match the configuration file.

Reviewer: Suggest to use `app plugin` since we use that in other places, e.g. https://grafana.com/docs/grafana/latest/administration/plugin-management/#app-plugins.

```suggestion
This allows app plugins to be configured and enabled/disabled using configuration files.

You can manage app plugins in Grafana by adding one or more YAML config files in the [`provisioning/plugins`]({{< relref "../setup-grafana/configure-grafana/#provisioning" >}}) directory. Each config file can contain a list of `apps` that will be updated during start up. Grafana updates each app to match the configuration file.
```

> > **Note:** This feature enables you to provision plugin configurations, not the plugins themselves. 

Reviewer: ```suggestion
> **Note:** This doesn't install any plugins, they must be installed separately. 
```

> > The plugins must already be installed on the grafana instance

Reviewer: ```suggestion
```
